deploy/0060_deploy_gems.ts

```diff
@@ -0,0 +1,19 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+import gems from '../data/gems';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts} = hre;
+  const {deploy} = deployments;
+
+  const {deployer} = await getNamedAccounts();
+  for (const gemName of gems) {
+    await deploy(`Gem_${gemName}`, {
+      contract: 'ERC20Token',
+      from: deployer,
+      log: true,
+      args: [gemName, gemName, deployer],
+    });
+  }
+};
+export default func;
```

Comment:
This should define tags (and dependencies if any) see : https://github.com/thesandboxgame/sandbox-private-contracts/blob/b14f22fcc2e270db7ab211cebf9b05da8c8d9d80/deploy/2000_deploy_permit.ts#L18-L20

Here it would simply be 
`func.tags = ["Gems"]`